runtime/centrifuge/src/migrations.rs

```diff
@@ -15,11 +15,7 @@ pub type UpgradeCentrifuge1022 = (
 	anemoy_pool::Migration,
 	add_wrapped_usdc_variants::Migration,
 	// Sets account codes for all precompiles
-	runtime_common::migrations::precompile_account_codes::Migration<
-		crate::Runtime,
-		{ crate::VERSION.spec_version },
-		1020,
-	>,
+	runtime_common::migrations::precompile_account_codes::Migration<crate::Runtime, 1020>,
```

Comment:
This will exclude the migration from Catalyst as the `last_version` is 1021 here. I'd rename `WITH_VERSION` to `MAX_VERSION` and set it to 1021 for Centrifuge runtime.